README.md

```diff
@@ -13,11 +13,24 @@ To start setting up the environment with BDD (Behavior Driven Development).
 * Install [Git version 2.21.0.windows.1](https://git-scm.com/download/win)
 
 ## Configuration
-List of required libraries in Project interpreter
-* Install [behave](https://www.jetbrains.com/help/pycharm/creating-feature-files.html) interpreter in PyCharm.
-* Install [requests](https://pypi.org/project/requests/)
-* Install [jsonschema](https://pypi.org/project/jsonschema/)
-* Install [Pytest](https://docs.pytest.org/en/latest/getting-started.html) to write unit tests
+This project was intended to work with python 3.8.1 A list of required libraries can be found on "requirements.txt" 
+file.
```

Comment:
Would it be in code line 16?

Reply:
yes, but I put it down so as not to pass the delimitation line.